spacelift/resource_api_key.go

```diff
@@ -0,0 +1,237 @@
+package spacelift
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/shurcooL/graphql"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/validations"
+)
+
+func resourceAPIKey() *schema.Resource {
+	return &schema.Resource{
+		Description: "`spacelift_api_key` represents a Spacelift API Key - " +
+			"a credential that can be used to authenticate with the Spacelift API " +
+			"from outside of Spacelift, typically for automation purposes.\n\n" +
+			"### WARNING\n\n" +
+			"** This resource manages API keys which are sensitive credentials. " +
+			"These keys will be saved to your state file. Ensure that you handle your state file securely. **",
```

Comment:
```suggestion
			"**This resource manages API keys which are sensitive credentials. " +
			"These keys will be saved to your state file. Ensure that you handle your state file securely.**",
```

Little markdown syntax fix:

<img width="1194" height="406" alt="Image" src="https://github.com/user-attachments/assets/122909e9-ec4f-474b-a58d-4b32350d41a0" />